docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
+available on PyPI, so we have to install them from RPMs.
+
+Install virtualenv::
+
+  > sudo pip install virtualenv
+
+Add your user to ``qemu`` group and vice versa::
+
+  > sudo usermod -a -G qemu "$USER"
+  > sudo usermod -a -G "$USER" qemu
+
+Add a polkitd policy to allow your user to connect to libvirtd::
+
+  > echo "polkit.addRule(function(action, subject) {
+    if (action.id == \"org.libvirt.unix.manage\" &&
+       subject.user == \"$USER\") {
+          return polkit.Result.YES;
+          }
+        }); " | sudo tee "/etc/polkit-1/rules.d/50-libvirt-$USER.rules"
+
+Enable nested virtualization(assuming Intel here)::
+
+ > echo "options kvm-intel nested=y" | sudo tee /etc/modprobe.d/kvm-intel.conf
+ > sudo bash -c 'modprobe -r kvm_intel && modprobe kvm_intel'
+
+Enable libvirtd::
+
+ > sudo bash -c 'systemctl enable libvirtd && systemctl start libvirtd'
+ > sudo bash -c 'systemctl enable virtlogd && systemctl start virtlogd'
+
+Update home directory permissions::
+
+ > chmod g+x "$HOME"
+
+We are going to create the following directory structure::
+
+ .
+ ├── data
+ │   ├── repo - lago repo directory
+ │   ├── store - lago template store directory
+ │   └── subnets - lago subnets lease files
+ ├── lago - lago git repository
+ └── venv-lago - venv installed modules
+
+Create the directories::
+
+ > mkdir -p "$HOME"/data/{repo,store,subnets}
+
+Overriding the ``subnet_lease_dir`` is still not supported, so we will
+have to create the directory under ``/var/lib/lago``::
+
+  > sudo mkdir -p /var/lib/lago
+  > sudo chown "$USER:$USER" /var/lib/lago
+
+Create a local ``.lago.conf`` file pointing to our new directory structure::
+
+ > cat > "$HOME/.lago.conf" << EOF
+   [lago]
+   template_store=/home/$USER/data/store
+   template_repos=/home/$USER/data/repo
+   subnet_lease_dir=/home/$USER/data/subnets
```

Comment:
Overriding `subnet_lease_dir` requires #298 :-) Once merged, the sentence above starting with `Overriding the subnet_lease_dir` could be removed.